app/models.py

```diff
@@ -96,3 +98,84 @@ def facet_plural(facet: str):
         facet_plural = facet
 
     return facet_plural
+
+
+# --------------------------------------------
+# Response model class for the knowledge panels
+# --------------------------------------------
+
+
+class ElementsForDataQualityAndLastEdits(BaseModel):
+    element_type: str
+    text_element: Optional[str] = None
```

Comment:
I would use a more generic term for this like TextFacet.

Also I would make ElementsForHungerGame and TextItemWikiData inherit it.

```python
class TextFacet(BaseModel):
    """Base facet containing text"""
    element_type: str
    text_element: Optional[str] = None
```

Reply:
Also in all the file we also want to be more informative and provide not only type, but also description, eventual example.

For that you need to use Field, see https://fastapi.tiangolo.com/tutorial/body-fields/

Reply:
Here look If I inherit the "Textfacet" model in "TextItemWikiData" as you said, then we get this issue again https://github.com/openfoodfacts/facets-knowledge-panels/pull/24#discussion_r963656050